SOAtestToMTM/README.md

```diff
@@ -61,79 +61,79 @@ Same structure in MTM
+* Open Sthe OAtestToMTM solution in Visual Studio
+* Update the missing Nuget Dependencies
+* Build the Solution
 
 ### Command Line
 TBA
 
 ## How to Run
-The executable requires the uri to Team Foundation Server and the valid credentials. A valid project name is also required to import SOAtest Result.
-In command prompt, run the executable with the following arguments:
+The MTM Importer executable requires the uri to Team Foundation Server and the valid credentials. A valid project name is also required to import SOAtest Result.
+In a command prompt, run the MTM Importer executable with the following arguments:
 ```
 MTMImporter.exe <TFS uri> <TFS username> <TFS password> <TFS domain> <TFS Project> <path to SOAtest report.xml>
 ```
 
-When import is completed without error, the following message will be displayed
+When the import is completed without error, the following message will be displayed:
 
 ```
 C:\Example>MTMImporter.exe http://tfs2013.parasoft.com:8080/tfs/DefaultCollection username password PARASOFT "Project A" "C:\Report\report.xml"
 Import completed
 ```
 
 
-## Pasring SOAtest results (report.xml)
+## Parsing SOAtest results (report.xml)
 
 ### Overview
-There are a lot of information in the report but we will be utilizing only several key elements.
+Report.xml contains a lot of information; here, we will focus only on several key elements.
 
-The example is using XMLReader for simplicity and efficiency reasons and expect elements in the following order (default order):
+Our example uses XMLReader for the sake of simplicity and efficiencys. It expects elements in the following order (default order):
 
 1. `ResultsSession` root element and its attributes to get general information for the test run, such as project, session tag, test start time and build number.
 2. `TestConfig` element for the configuration and user.
 3. `ExecutedTestsDetails` element and its children to gather test result
 4. `FuncViols` element for test failures.
 
 ##### ResultsSession
-ResultsSession is the root element in SOAtest report.xml and the following attributes will be used to populate infomration for a MTM Test Run.
-* `buildId` will be used for build number
-* `project` will be used for project name
-* `tag` will be used for for test environment. Note: tag is a user defiend field "Session Tag" in SOAtest, it can be used to group multiple reports, in this case, a test environment.
-* `time` will be used for time start time, we will have to calculate the test end time based of this.
+ResultsSession is the root element in SOAtest report.xml. The following attributes will be used to populate information for a MTM Test Run.
+* `buildId`: For build number
+* `project`: For project name
+* `tag`: For test environment. Note that tag is a user-defined field ("Session Tag" in SOAtest). It can be used to group multiple reports-- in this case, a test environment.
+* `time`: For time start time, we will have to calculate the test end time based of this.

```

Comment:
I think you can just replace the description for `time` to 
For test start time